utils.go

```diff
@@ -92,6 +92,21 @@ func revisePidFile(context *cli.Context) error {
 	return context.Set("pid-file", pidFile)
 }
 
+// reviseRootDir convert the root to absolute path
+func reviseRootDir(context *cli.Context) error {
+	rootFile := context.GlobalString("root")
```

Comment:
`--root` isn't a file, the variable should be called `root`.

Reply:
Should *rootFolder* be more appropriated here?

Reply:
Just `root` (or `rootDir` if you really want to use two words).

Reply:
ok, kept root in the last commit.